user_groups/user_groups_schemas.md

```diff
@@ -20,4 +21,13 @@ type UserGroup = {
   ; name: string
   ; description: string
 }
+```
+
+## User
+
+```typescript
+type User = {
```

Comment:
@WMKhant  This should go to the user schemas file and this is a UserListItem. We do not have a name in the user, so we also need to change the API to get the list of users to return { id: "", loginName: "" }.

Reply:
this has been remove and the User list item is updated with loginName @myfoxtail